x/auth/tx/service_test.go

```diff
@@ -194,7 +194,7 @@ func (s IntegrationTestSuite) TestGetTxEvents_GRPC() {
 		{
 			"with multi events",
 			&tx.GetTxsEventRequest{
-				Events: []string{"message.action=send", "message.module=bank"},
+				Events: []string{"message.action='send'", "message.module='bank'"},
```

Comment:
is `'` required?

Reply:
It's required by tendermint. The old legacy REST also requires it.

Somehow, in the gRPC handler, we added a logic saying that "if there's no `'`, we add a `'`". I think people are used to `'`, so I'm proposing to do exactly what tendermint expects.